persistence/chain.json

```diff
@@ -12,7 +12,7 @@
   "key_algos": [
     "secp256k1"
   ],
-  "slip44": 750,
```

Comment:
Still registered as 750 in https://github.com/satoshilabs/slips/blob/master/slip-0044.md. Is this planned to be removed from that list?

Reply:
it is planned, we have changed it in code to 118, but we currently still support 750 so that migration happens, wondering when is the good time to change the slip